UnityProjectDP/Assets/Scripts/Visualization/ClassDiagram/ClassEditor.cs

```diff
@@ -382,8 +549,53 @@ public static void AddAttribute(string targetClass, Attribute attribute, bool fr
             AddTmProAttribute(classInDiagram.VisualObject, attributeText);
         }
 
+        public static bool UpdateAttribute(string targetClass, string oldAttribute, Attribute newAttribute)
+        {
+            var classInDiagram = DiagramPool.Instance.ClassDiagram.FindClassByName(targetClass);
+            if (classInDiagram == null)
+            {
+                return false;
+            }
```

Comment:
dohodnime sa ako s tymi zatvorkami. Tu ich amme a na r. 561 uz nie su. Minimalne pri returnoch by som ich ja nedaval.
```suggestion
                return false;
```